Makefile

```diff
@@ -36,7 +33,18 @@ $(OPERATOR): vendor
 
 .PHONY: test
 test:
-	go test $(GO_FLAGS) $(GO_TEST_FLAGS) ./pkg/apis/... ./pkg/controller/...
+	GO111MODULE=on ginkgo \
```

Comment:
Could you modify the `Makefile` to automatically install `ginko` ?

Reply:
@baijum how do you want such deps to be installed for openshift ci?

Reply:
We can modify `openshift-ci/Dockerfile.tools` file install the binary deps.

Reply:
Would you be able to make a PR to this branch, Baiju?

Reply:
I have sent a separate PR: https://github.com/redhat-developer/build/pull/107

Reply:
I just rebased on top of master, which have pr 107.